.travis.yml

```diff
@@ -21,23 +21,20 @@ env:
     - PIP_WHEEL_DIR=$HOME/.cache/pip/wheels
     - PIP_FIND_LINKS=file://$HOME/.cache/pip/wheels
 
-python:
-    - 2.6
-    - 2.7
-
+  matrix:
+    - TOX_ENV=py26
+    - TOX_ENV=py27
```

Comment:
I think this is because we also want to set global environment variables, and so the 'env' section becomes a mapping rather than a list. e.g. 

```
env = {'global': ['PIP_WHEEL_DIR=$HOME/.cache/pip/wheels', ... ],
           'matrix': ['TOX_ENV=py26', 'TOX_ENV=py27']}

vs

env = ['TOX_ENV=py26', 'TOX_ENV=py27']
```

http://docs.travis-ci.com/user/environment-variables/#Global-Variables